modules/xmpp/ChatRoom.js

```diff
@@ -613,6 +614,36 @@ export default class ChatRoom extends Listenable {
                     this.eventEmitter.emit(XMPPEvents.BRIDGE_DOWN);
                 }
                 break;
+            case 'conference-properties':
```

Comment:
These should be accepted if they come from jicofo (`member.isFocus`), no?

Reply:
Yes, you're right. I pushed a fix.